src/roiextractors/multiimagingextractor.py

```diff
@@ -33,12 +33,18 @@ def __init__(self, imaging_extractors: List[ImagingExtractor]):
 
         self._start_frames, self._end_frames = [], []
         num_frames = 0
+        times = []
         for imaging_extractor in self._imaging_extractors:
             self._start_frames.append(num_frames)
             num_frames = num_frames + imaging_extractor.get_num_frames()
             self._end_frames.append(num_frames)
+            if getattr(imaging_extractor, "_times") is not None:
+                times.extend(imaging_extractor._times)
         self._num_frames = num_frames
 
+        if times:
+            self.set_times(times=times)
```

Comment:
or do the assertion that the length matches and:
`self._times = times `

Reply:
Hmmm good question, I'll look into it and get back to you

Reply:
Seems from the core logic of https://github.com/catalystneuro/roiextractors/blob/master/src/roiextractors/imagingextractor.py#L71-L88 we may want to pre-scan the list of extractors to see if any one of them has times set (`any((hasattr(extractor, "_times") for extractor in imaging_extractors))`) and if even a single one does, we'll need to assemble an entire `_times` vector for the `MultiImagingExtractor` as well, assemebled by either appending/concatenating the `_times` field if available in a sub extractor, or by getting `frame_to_time` artificially using its sampling frequency.